test_utils_helpers.py

```
@@ -0,0 +1,518 @@
+        
+        # Chunk the keys for processing
+        keys = list(parsed_data.keys())
+        key_chunks = chunk_list(keys, 2)
+        
+        assert len(flat_data) == 6  # All nested keys flattened
+        assert parsed_data == flat_data  # Round-trip successful
+        assert len(key_chunks) == 3  # 6 keys chunked by 2
+
+
+if __name__ == "__main__":
+    pytest.main([__file__, "-v"])
```

Pylint (3.3.7)

[convention] Final newline missing

(C0304)